### DISCUSSION TITLE
Twin model failed during instantiation

### DISCUSSION THREAD

**Discussion by ctrwtskm** [ansys/pytwin#205] [General]
Hi, 
I am currently working with and still new to PyAnsys, particularly PyTwin. 
I found and issue that I can't instantiate my model (it has dynamic rom). It gives the same error when I use PyTwin version 0.7.0 and 0.8.dev0 (pic below).
But when I use PyTwin version 0.6.0 it can work well.
![image](https://github.com/user-attachments/assets/7eed8a2a-9b08-429e-86f9-9ae50b88f4d1)

Can anyone suggest on how I resolve this issue in the recent version of PyTwin?
Please also find the twin file used:
[Twin_displacement_direction.zip](https://github.com/user-attachments/files/17725527/Twin_displacement_direction.zip)

**Comment by chrpetre** [comment]
hello @ctrwtskm, there is an issue with Dynamic ROM builder for this particular model (that issue has been reported internally to be fixed in a newer version of Twin Builder). I have attached a corrected twin file which should not show the problem anymore with current version of pytwin. Feel free to re-open if you still experience issues with that model.
[Twin_displacement_direction_Ansys.zip](https://github.com/user-attachments/files/17731755/Twin_displacement_direction_Ansys.zip)
